cordova-plugin-appcenter-shared/src/android/AppCenterShared.java

```diff
@@ -29,6 +31,14 @@ static void configureAppCenter(Application application, CordovaPreferences prefe
         AppCenter.setWrapperSdk(wrapperSdk);
         AppCenter.configure(application, AppCenterShared.getAppSecret(preferences));
 
+        final int logLevel = preferences.getInteger(LOG_LEVEL_KEY, Integer.MIN_VALUE);
+        if (logLevel > 1 && logLevel < 8) {
+            AppCenter.setLogLevel(logLevel);
+            AppCenterLog.error(AppCenterLog.LOG_TAG, "We log" + logLevel);
```

Comment:
1) why .error?
2) there is no log in the same place in objc file.

Reply:
Oops, that's debug line 😄